lib/Backend/arm64/LowerMD.cpp

```diff
@@ -5125,6 +5129,7 @@ LowererMD::EmitLoadFloatFromNumber(IR::Opnd *dst, IR::Opnd *src, IR::Instr *inse
         tempSymOpnd = IR::SymOpnd::New(this->m_func->tempSymDouble, TyFloat64, this->m_func);
         instr = IR::Instr::New(Js::OpCode::LEA, reg3Opnd, tempSymOpnd, this->m_func);
         insertInstr->InsertBefore(instr);
+        Legalize(instr);
```

Comment:
Lowerer::InsertLea instead

Reply:
Changed places to use InsertAdd/InsertLea as appropriate where possible.